coalib/processes/BearRunning.py

```diff
@@ -5,8 +5,8 @@
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
+from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
```

Comment:
We have a `PyImportSortBear`, however it doesn't support our import sorting standards yet :/ (I think there are also some upstream problems)